### PR TITLE
create composant admin and composant popover

### PR DESCRIPTION
**Pull request by illshaad**

### CODE REVIEW

File: client/composants/popover/Popover.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
+import * as React from "react";
+import Popover from "@mui/material/Popover";
+import Typography from "@mui/material/Typography";
+import Button from "@mui/material/Button";
+import { FlexContainer, Card, ContainerInformation } from "../../styles/global";
+
+export default function PopoverComposant() {
```
Comment (Frozenix94) [inline client/composants/popover/Popover.js:7]: Composant in name is redondant here :) this one could be use all over your app ?
If yes, you can rename this one Popover only and rename your import into PopoverMui :)

File: client/composants/popover/Popover.js [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+  const handleClick = (e) => {
+    setAnchorEl(e.currentTarget);
+  };
+  const handleClose = () => {
+    setAnchorEl(null);
+  };
+
+  const open = Boolean(anchorEl);
+  const id = open ? "simple-popover" : undefined;
```
Comment (Frozenix94) [inline client/composants/popover/Popover.js:18]: Why did you do this ? you cannot set id by default ?

If not you can do this directly in component

File: client/composants/popover/Popover.js [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+  return (
+    <div>
+      <Button aria-describedby={id} variant="contained" onClick={handleClick}>
+        Les Avis
+      </Button>
+      <Popover
+        id={id}
```
Comment (Frozenix94) [inline client/composants/popover/Popover.js:26]: follow last comment : 
```suggestion
        id={open ? "simple-popover" : undefined}
```

File: client/styles/global.js [hunk 4]
```diff
@@ -51,17 +51,18 @@ export const FlexContainer = styled.div`
   flex-direction: ${({ flexDirection }) => flexDirection || null};
   justify-content: space-around;
   align-items: center;
-  margin-bottom: 20px;
+  margin: ${({ margin }) => margin || "20px 20px"};
```
Comment (Frozenix94) [inline client/styles/global.js:54]: here you set margin 20px anyway ? you can simply pass 20px one time :)
```suggestion
  margin: ${({ margin }) => margin || "20px"};
```

File: client/composants/popover/Popover.js (hunk 2 above)
Comment (illshaad) [reply client/composants/popover/Popover.js:18]: I admit that this part of the code I did not understand well. It is component of materiel ui

Comment (illshaad) [reply client/composants/popover/Popover.js:18]: I would like to see it with you. When you have the time of course xD

**Pull request merged**